source/glossary.rst

```diff
@@ -93,6 +93,22 @@ The Operator options glossary
 
       :ref:`pmm.image<pmm-image>`
 
+   imagePullPolicy
```

Comment:
@spron-in  I have re-check and we do not support it for now at all. I am going to create task about it,